exonum/src/blockchain/mod.rs

```diff
@@ -449,6 +449,15 @@ impl BlockchainMut {
             .expect("Cannot update transaction pool");
     }
 
+    /// Performs several shallow checks that transaction is correct.
+    ///
+    /// Returned `Ok(())` value doesn't necessarily mean that transaction is correct and will be
+    /// executed successfully, but returned `Err(..)` value means that this transaction is
+    /// **obviously** incorrect and should be declined as early as possible.
+    pub(crate) fn check_tx(&self, tx: &Verified<AnyTx>) -> Result<(), ExecutionError> {
```

Comment:
`is_processable` doesn't seem good to me, since this transaction can be **not** processable even with `Ok(())` returned from this method.
And since this method will be extended in the future, I believe we'll be able to figure out a better name once it'll happen.